lib/registration.pm

```diff
@@ -233,6 +234,22 @@ sub remove_suseconnect_product {
     script_retry("SUSEConnect -d -p $name/$version/$arch $params", retry => 5, delay => 60, timeout => 180);
 }
 
+=head2 ssh_remove_suseconnect_product
+
+    ssh_remove_suseconnect_product($name, [$version, [$arch, [$params]]]);
+
+wrapper for suseconnect -d $name over ssh.
+=cut
+sub ssh_remove_suseconnect_product {
+    my ($remote, $name, $version, $arch, $params) = @_;
+    assert_script_run "sftp $remote:/etc/os-release /tmp/os-release";
```

Comment:
what about our lovely  "fingerprint have changed " error , I assume we will start getting it here in some cases ?

Reply:
I didn't notice it lately. Plus it's in my opinion separate problem to solve.

Reply:
ok